ivy/functional/frontends/paddle/tensor/stat.py

```diff
@@ -5,8 +5,12 @@
     to_ivy_arrays_and_back,
 )
 
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def mean(x, axis=None, keepdims=False, out=None):
```

Comment:
Can you also change the argument `input` to `x` in the test for this function?

Reply:
Sure!